### PR TITLE
Migrated templatetags from py-nepali project

### PR DESCRIPTION
**Pull request by aj3sh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,92 @@
 # django-nepali
+
 A django package on top of 'nepali' a python package which supports nepali date time, time conversion, etc on django projects.
```
Comment (sugat009) [inline README.md:3]: Let's add a hyperlink on the `nepali` package for better reachability

File: django_nepali/templatetags/nepalidatetime.py [hunk 2]
```diff
@@ -0,0 +1,164 @@
+import datetime
+from typing import Optional, Union
+
+from django import template
+from django.utils import timezone
+
+from nepali.datetime import nepalidate as _nepalidate
+from nepali.datetime import nepalidatetime as _nepalidatetime
+from nepali.datetime import nepalihumanize as humanize
```
Comment (sugat009) [inline django_nepali/templatetags/nepalidatetime.py:9]:
```suggestion
from nepali.datetime import (
    nepalidate as _nepalidate,
    nepalidatetime as _nepalidatetime,
    nepalihumanize as humanize
)
```

**Review by sugat009 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (sugat009) [reply README.md:3]: .

**Review by sugat009 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (aj3sh) [reply README.md]: Updated

**Review by sugat009 — APPROVED** [review]

**Pull request merged**